code/src/components/CustomCheckbox.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+import { CheckBoxContainer, HiddenCheckBox, StyledCheckbox, Icon} from 'styling/CheckBoxStyling';
```

Comment:
I didnt even know we had a ready checkbox. I spent so much time making my own. You have to show me how this works.

Reply:
Karan, I think Jessica created her own little library of styled components 🤩